lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -164,7 +164,7 @@ protected function doDelete($id)
     {
         $filename = $this->getFilename($id);
 
-        return @unlink($filename) || ! file_exists($filename);
```

Comment:
It would be very helpful in debugging to have the error if unlink fails for something other than the file not existing.

Out of curiosity how could `unlink()` fail if it actually deleted the file?